upup/pkg/kutil/rollingupdate_cluster.go

```diff
@@ -266,8 +290,26 @@ func (n *CloudInstanceGroup) RollingUpdate(cloud fi.Cloud, force bool, interval
 			return fmt.Errorf("error deleting instance %q: %v", instanceID, err)
 		}
 
-		// TODO: Wait for node to appear back in k8s
+		// Wait for new EC2 instances to be created
 		time.Sleep(interval)
+
+		// Wait until the cluster is happy
+		// TODO: do we need to respect cloud only??
+		for i := 0; i < retries; i++ {
+
+			_, validateDidNotPass := validate.ValidateCluster(u.Node.ClusterName, instanceGroupList, k8sClient)
+
+			if validateDidNotPass != nil {
+				glog.V(2).Infof("Unable to validate kuberetes cluster %s, %v.", u.Node.ClusterName, validateDidNotPass)
+				time.Sleep(interval)
+			} else {
+				glog.V(2).Infof("Cluster %s is validated proceeding with next step in rolling update",
+					u.Node.ClusterName)
+				break
+			}
+		}
+
+		// TODO: what do we do if validation fails?
```

Comment:
Presumably stop, unless we're running in `--force` (or another flag if we think that flag is orthogonal)